example/basic/main.go

```diff
@@ -82,12 +82,26 @@ func main() {
 	_ = c.Start(context.Background())
 
 	meter := c.Meter("otel.dynatrace.com/basic")
-	vr := metric.Must(meter).NewFloat64Histogram("golang_histogram")
+	vr, err := meter.SyncFloat64().Histogram("golang_histogram")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Was `Must` removed?

Reply:
It was removed on opentelemetry, there's no equivalent now. All examples available do this manual error checking now.